### DISCUSSION TITLE
请教怎么设置前置代理（链式代理）

### DISCUSSION THREAD

**Discussion by Yamazaki-wu** [SagerNet/SagerNet#454] [General]
请问一下Sager Net怎么设置前置代理？就是我有一个机场（公益）节点，联通线路连接了很快，但是电信连接就一般般。
另外在联通线路下搭建了个socks5，能否在Sager Net中设置让访问先走socks再连接机场（公益）节点？
PC上的QV2ray就能设置并正常使用socks作为前置。

**Comment by Nelocompne** [comment]


**Reply by Yamazaki-wu** [reply]
可否告知具体怎么操作么？或者有教程也行。我不是很在行，但可以学着尝试。

**Comment by Yamazaki-wu** [comment]
根据你超详细的步骤，发现挺简单的，非常感谢！

Minn ***@***.***> 于2022年5月19日周四 00:26写道：

**Comment by tinaqh** [comment]
你好，能告诉一下怎么设置前置代理么
